src/SFML/Window/OSX/SFOpenGLView+mouse_priv.h

```diff
@@ -27,6 +27,7 @@
 // Headers
 ////////////////////////////////////////////////////////////
 #include <SFML/Window/Mouse.hpp>
+#import <SFML/Window/OSX/SFOpenGLView.h>
```

Comment:
Isn't this doing more than just formatting?

Reply:
Fixing compiler errors caused by clang-format reordering headers.